Python/instrumentation.c

```diff
@@ -412,13 +414,14 @@ dump_instrumentation_data(PyCodeObject *code, int star, FILE*out)
     assert(test); \
 } while (0)
 
-bool valid_opcode(int opcode) {
+bool valid_opcode(int opcode)
```

Comment:
Since we're modifying this, might as well to follow PEP 7
```suggestion
bool
valid_opcode(int opcode)
```

Reply:
```suggestion
static bool
valid_opcode(int opcode)
```

Reply:
`get_base_opcode_best_attempt()` in line 338 is also file local.